src/joinUrl.test.ts

```diff
@@ -0,0 +1,45 @@
+import { joinUrl } from './joinUrl';
+
+const expectedOutput = 'http://example.com/abc/xyz';
+
+describe('joinUrl', () => {
+	it('prevents double slashes correctly', () => {
+		expect(joinUrl('http://example.com/', '/abc/', '/xyz/')).toBe(
+			expectedOutput,
```

Comment:
Totally personal preference but I'd move the `expectedOutput` const into each test case rather than sharing (or even inline the string). I love not having to bounce around the test file to fully understand a test, and I'd argue it makes it easier to change a single test down the line without having to worry about impact on other tests.

Reply:
I wholeheartedly agree, tests should be as simple as possible. Thanks for pointing this out, I'll update